include/arc_utilities/dijkstras.hpp

```diff
@@ -388,9 +423,10 @@ namespace arc_dijkstras
         Graph()
         {}
 
+
         uint64_t SerializeSelf(
-                std::vector<uint8_t>& buffer,
-                const std::function<uint64_t(const NodeValueType&, std::vector<uint8_t>&)>& value_serializer) const
+            std::vector<uint8_t>& buffer,
```

Comment:
These whitespace changes appear to make these 2 functions have different formatting from the rest of the file. Any particular reason for the change?

Reply:
Good catch
Editor preferences. I will return to original indentation

Reply:
Ping.